saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/web/Saml2WebSsoAuthenticationFilter.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.security.saml2.provider.service.servlet.filter;
+package org.springframework.security.saml2.provider.service.web;
```

Comment:
I appreciate you being proactive here, but moving things will take a bit more work so that things stay backward compatible. While new filters like `Saml2MetadataFilter` should go into `web`, let's please wait on moving existing filters until https://github.com/spring-projects/spring-security/issues/8819

Reply:
I cannot move new filters without moving the rest or changing the visibility of *Utils classes to public. Are you OK with the second one?

Reply:
No, the Utils classes should not be made public. I'd recommend inlining that support, as was done with `DefaultSaml2AuthenticationRequestContextResolver`. Once everything gets moved, then the code can be changed to call the Utils classes again.